qiskit/pulse/exceptions.py

```diff
@@ -27,11 +27,11 @@ def __str__(self):
         return repr(self.message)
 
 
-class BackendNotSet(PulseError):
+class BackendNotSetError(PulseError):
     """Raised if the builder context does not have a backend."""
 
 
-class NoActiveBuilder(PulseError):
+class NoActiveBuilderError(PulseError):
     """Raised if no builder context is active."""
```

Comment:
This is an API break - we can't do this.

Reply:
This has been reverted.